.github/workflows/ci-zod-openapi.yml

```diff
@@ -12,13 +12,17 @@ on:
 jobs:
   ci:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        hono-version: ['hono@^4.0.0']
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
           node-version: 20.x
+      - run: yarn add ${{ matrix.hono-version }} --mode=update-lockfile
       - run: yarn workspaces focus hono-middleware @hono/zod-openapi
-      - run: yarn workspace @hono/zod-openapi build
+      - run: yarn workspaces foreach --topological --recursive --from @hono/zod-openapi run build
```

Comment:
Recessively build the dependencies of the current workspace

This is specifically for `@hono/zod-openapi` as it has a dependency on `@hono/zod-validator` which needs building first